app/users/users.go

```diff
@@ -258,3 +262,22 @@ func (us *UserService) PromoteGuestToUser(user *model.User) error {
 func (us *UserService) DemoteUserToGuest(user *model.User) (*model.User, error) {
 	return us.store.DemoteUserToGuest(user.Id)
 }
+
+func (us *UserService) ToggleEmailNotifications(userId string, action string) error {
```

Comment:
"Toggle" typically refers to toggling the state from one to the other, without needing to know the current state. In the use cases above you want to ensure the state is explicitly enabled or disabled.

What do you think of either of these options:
a) two methods, `EnableEmailNotifications(userID string)` and `DisableEmailNotifications(userID string)`, or
b) `SetEmailNotifcationsState(userID string, enabled bool)`

Reply:
@mkraft  When you put it that way, it makes sense to change it to SetEmailNotifcationsState. Thanks a lot.